site/data/examples.yml

```diff
@@ -33,3 +33,9 @@
   examples:
     - name: "Masonry"
       description: "Combine the powers of the Boosted grid and the Masonry layout."
+
+- category: Forms
```

Comment:
Will we have several forms ? If no, I think this section could be merged with snippets ? Or maybe just more inclusive like `Layout` or something like that.

Reply:
Relative to second comment

+

Should we turn this section into a 'Branded pages examples'? @julien-deramond ?

Reply:
Thinking out loud but maybe we could have the following:
- **Snippets** (as in Bootstrap) would contain the variants of our components that do not fit in directly in the documentation 
  - Footers + Cards + Stickers + Tags + Title bars + Navbars
- **Custom components** (as in Bootstrap) would be the components that can be created only with utilities AND the templates
  - Navbar sticky
- (Navbars could be removed since we have only 2 examples)
- **Pages**: complete reusable pages that we can find in the DSM
  - Form example

In this case, Forms would be treated as a Page and would include a navbar and footer since the reference is https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53.
This page should contain the same things (without the Phone Number form control that should be added in this example page when we will have this component in Boosted - add it to the issue)

Thoughts?